src/script/components/manifest-options.ts

```diff
@@ -857,10 +857,10 @@ export class AppManifest extends LitElement {
     const input = <HTMLInputElement | HTMLSelectElement>event.target;
     const fieldName = input.dataset['field'];
 
-    console.log('input.value', input.value, 'fieldName', fieldName);
-
     if (this.manifest && fieldName && this.manifest[fieldName]) {
-      this.updateManifest({ [fieldName]: input.value });
+      // to-do Justin: Figure out why typescript is casting input.value to a string
+      // automatically and how to cast to a better type that will actually compile
+      this.updateManifest({ [fieldName]: (input.value as any).code ? (input.value as any).code : input.value });
```

Comment:
Lack of type here makes this code harder to read.

Also, we might be able to simplify as, `input.value?.code || input.value`